pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt.go

```diff
@@ -134,8 +146,8 @@ func (handler DeviceCommandHandlerMQTT) commandHandleFunc() http.HandlerFunc {
 
 		if reqType == http.MethodGet {
 			returnMessage := ReturnBody{
-				MQTTMessage:   mqttMessageStr,
-				MQTTTimestamp: mqttMessageReceiveTimestamp.String(),
+				MQTTMessage:   mqttMessageStr[handler.HandlerMetaData.properties.MQTTTopic],
+				MQTTTimestamp: mqttMessageReceiveTimestamp[handler.HandlerMetaData.properties.MQTTTopic].String(),
```

Comment:
mqttMessageReceiveTimestamp[handler.HandlerMetaData.properties.MQTTTopic] would this be nil?

Reply:
no，mqttMessageReceiveTimestamp is a global variable that has been initialized